compiler/rustc_middle/src/query/mod.rs

```diff
@@ -431,6 +431,11 @@ rustc_queries! {
         desc { "computing `#[expect]`ed lints in this crate" }
     }
 
+    query lints_that_can_emit(_: ()) -> &'tcx Lrc<(FxIndexSet<String>, FxIndexSet<String>)> {
+        arena_cache
```

Comment:
If we have it on an arena, there is no need to put it inside a `Lrc`. It's already being made a pointer.

Reply:
Changed to `FxIndexSet` without `Lrc`